src/main/kotlin/io/github/inductiveautomation/kindling/statistics/categories/DatasourcesByDriverType.kt

```diff
@@ -0,0 +1,60 @@
+package io.github.inductiveautomation.kindling.statistics.categories
+
+import io.github.inductiveautomation.kindling.statistics.GatewayBackup
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+
+@Suppress("unused")
+class DatasourcesByDriverType(override val gwbk: GatewayBackup) : StatisticCategory() {
+    override val name = "DatasourcesByDriverType"
+
+    val mariadb by statistic { getDriverCount(DatabaseDriver.MARIADB) }
+    val sqlite by statistic { getDriverCount(DatabaseDriver.SQLITE) }
+    val mysql by statistic { getDriverCount(DatabaseDriver.MYSQL) }
+    val sqlserver by statistic { getDriverCount(DatabaseDriver.SQLSERVER) }
+    val oracle by statistic { getDriverCount(DatabaseDriver.ORACLE) }
+    val postgresql by statistic { getDriverCount(DatabaseDriver.POSTGRESQL) }
+
```

Comment:
I really like the implementation of this/how it looks in code...
But it's gotta be _way_ less efficient than a single query that then gets `groupBy`ed

Reply:
We can probably make this a `PreComputedStatisticCategory` to just run a select all statement